### PR TITLE
[AVFAudio] Xcode 10.2 Beta 2

### PR DESCRIPTION
**Pull request by chamons**

### CODE REVIEW

File: tests/monotouch-test/monotouch-test.csproj [hunk 1]
```diff
@@ -201,6 +201,7 @@
     <Compile Include="..\common\ConditionalCompilation.cs">
       <Link>ConditionalCompilation.cs</Link>
     </Compile>
+    <Compile Include="AVFoundation\FoureCCTest.cs" />
```
Comment (spouliot) [inline tests/monotouch-test/monotouch-test.csproj]: I think it was covered above with the 
`<Compile Include="**\*.cs" Exclude="obj\**">`

Comment (chamons) [reply tests/monotouch-test/monotouch-test.csproj]: Good point. It was auto added by VSfM and I didn't see that when reviewing.

**Comment by chamons** [comment]
build

**Comment by dalexsoto** [comment]
Finally some results, it seems that we are missing an entry in xtro

```
!unknown-native-enum! AVAudioSessionPromptStyle bound
```

**Comment by spouliot** [comment]
@dalexsoto this happens because `AVAudioSessionPromptStyle` has no availability information in headers. However it's only exposed on iOS/tvOS/watchOS (not macOS) so @chamons copied the same information (which is perfect).

However xtro/sharpie does not know that (not in the headers) so xtro must ignore that type for macOS.

**Comment by dalexsoto** [comment]
@spouliot agreed, I'll fix this 👍

**Pull request merged**